Interfaces/GCHP/Chem_GridCompMod.F90

```diff
@@ -409,6 +409,12 @@ SUBROUTINE SetServices( GC, RC )
     LOGICAL                       :: useCFMaskFile
 #endif
 
+    ! Manual internal state entries
```

Comment:
These updates are necessary to add the extra fields to the GCHP and/or GEOS internal state (and to the restart file).